db/migrate/20230210105741_add_assigned_to_regional_caseworker_team_to_projects.rb

```diff
@@ -0,0 +1,5 @@
+class AddAssignedToRegionalCaseworkerTeamToProjects < ActiveRecord::Migration[7.0]
+  def change
+    add_column :projects, :assigned_to_regional_caseworker_team, :boolean, default: false
```

Comment:
Do we need to migrate any data at this point? Happy to do so on the console as the number of project we have is so low.

Reply:
We're going to do this manually via the console after deployment.